### PR TITLE
driver plug-in should support non-DRM based device node

### PR DESCRIPTION
**Pull request by maxzhen**
#### Problem solved by the commit
In the future, the driver plug-in will need to support:
- user driver which is under Accel (vs DRM) framework, hence has a different device node path
- mgmt driver which is not called xclmgmt, hence has a different device node path

This pull request makes changes in driver plug-in to make it more flexible for the device node path / name.

### CODE REVIEW

File: src/runtime_src/core/pcie/linux/pcidev.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright (C) 2016-2020 Xilinx, Inc
-// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+// Copyright (C) 2016-2020, 2023 Xilinx, Inc
```
Comment (stsoe) [inline src/runtime_src/core/pcie/linux/pcidev.cpp:2]: I don't usually update Xilinx Inc on copyright these days.  Are we supposed to?

**Review by stsoe — APPROVED** [review]

**Comment by dayeh-xilinx** [comment]
retest this please.

File: src/runtime_src/core/pcie/linux/pcidev.cpp (hunk 1 above)
Comment (maxzhen) [reply src/runtime_src/core/pcie/linux/pcidev.cpp:2]: Oh, no. It's automatically done by my vim script. I need to fix it. We should update the AMD one.

Comment (keryell) [reply src/runtime_src/core/pcie/linux/pcidev.cpp]: That is a great idea to have such a script anyway!

**Pull request merged**